app/elements/candidates/invite-candidates.html

```diff
@@ -51,25 +55,21 @@
 					type: Array,
 					notify: true
 				},
-				challenge: {
-					type: Object,
-					notify: true
-				},
 				candidates: {
 					type: Array,
 					notify: true,
 					value: []
 				}
 			},
-			toggle: function(){
+			toggle: function () {
 				this._clearDialog();
 				this._addCandidate();
 				this.$.dialog.toggle();
 			},
-			inviteCandidates: function(){
-				for (var i = 0; i < this.candidates.length; i++){
+			inviteCandidates: function () {
+				for (var i = 0; i < this.candidates.length; i++) {
 					var candidate = this.candidates[i];
-					if (candidate.Removed !== undefined && candidate.Address !== '' && candidate.Challenge !== undefined){
+					if (candidate.Removed === undefined && candidate.Address !== '' && candidate.Challenge !== undefined) {
```

Comment:
Having a boolean that is `undefined` is not nice in any case, because most of the time you excpect `true` or `false`.

However, you could use `if (!!candidate.Removed)` which will be `true` if `candidate.Removed === true` and `false` if `candidate.Removed === false || candidate.Removed === undefined || candidate.Removed === null || ...`.